plugins/tekton/src/components/common/Table.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {
+  Table as BackstageTable,
+  TableColumn,
+} from '@backstage/core-components';
+
+type TableProps<D = any> = Partial<D> & {
+  tableData: any[];
+  columns: TableColumn[];
+  title?: string;
+  subTitle?: string;
+};
+
+export const Table = ({ tableData, columns }: TableProps) => {
```

Comment:
do we need this component now or can the caller directly use `Table` from backstage?